### ISSUE TITLE
Smallcase `patch` is making apis fail

### ISSUE THREAD

**Issue by sriniously** [scalar/scalar#4163]
### What happens?

I am using the html example in my go app.

```
<!doctype html>
<html>
  <head>
    <title>API Reference</title>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
  </head>
  <body>
    <script id="api-reference" data-url="/static/openapi.json"></script>
    <script src="https://cdn.jsdelivr.net/npm/@scalar/api-reference"></script>
  </body>
</html>
```

And patch requests are going as small-case, because of that in my golang echo app, the route is not matching.

![image](https://github.com/user-attachments/assets/3456cb88-91e7-449c-8403-f04a7eeb3064)

### What did you expect to happen?

The `patch` method should go as uppercase.

**Comment by antlio** [comment]
hey @hellskater thanks for reporting! this has been recently patched (pun intended) by @Huliiiiii #4159 and should be released soon

**Comment by hellskater** [comment]
Haha, thanks @antlio
